components/docs-chef-io/content/automate/ha_deploy_bareinfra.md

```diff
@@ -0,0 +1,119 @@
+
+1. Make the following changes in `config.toml` file by opening the file in a editor. For example, `vi config.toml`.
+
+   1. Specify the `ssh username` and the `ssh key file path`. The ssh key must reside in bastion host.
+   1. Ensure `ssh_key_pair_name` and `ssh key file path` have same value.
+   1. Assign permission to the *ssh key file* by running command, `chmod 400 /root/.ssh/id_rsa`.
+   1. Specify the number of nodes for the Chef Automate and Chef Infra server clusters. By default, the deployment takes value `1`.
+   1. Ensure not to modify the cluster number value as `3` for PostgreSQL and ElasticSearch.
+   1. Ensure the instance type supports the respective AWS region.
+   1. Add load balancer certificate details for automate and chef-server. For example, as shown below:
+
+      <!-- automate_lb_certificate_arn = "arn:aws:acm:ap-south-1:510367013858:certificate/1aae9fce-60df-4791-9bec-ef6a0f723f3e"
+      <!-- chef_server_lb_certificate_arn = "arn:aws:acm:ap-south-1:510367013858:certificate/1aae9fce-60df-4791-9bec-ef6a0f723f3e" --> -->
+
+      {{< figure src="/images/automate/ha_bare_lb.png" alt="Load Balancer Details">}}
+
+   1. Setup the secrets management key and the required passwords. The default location for the secrets key and secret storage is set in the *config.toml* file. The default location for the key is `/etc/chef-automate/secrets.key` and the secret store file is in `/hab/a2_deploy_workspace/secrets.json`.
+
+{{< figure src="/images/automate/ha_bare_chef_automate_configtoml_file.png" alt="Chef Automate Bare Infra `config.toml` file">}}
+
+1. Type the command, `./chef-automate deploy` and press **Enter**. This command installs the latest deployment package and deploys (by provisioning with terraform) airgap bundles on the created infrastructure. The deployment procedure creates the *HAB* user by default.
+
+{{< figure src="/images/automate/ha_bare_chef_automate_complete.png" alt="Chef Automate Bare Infra Deployment Confirmation">}}
+
+1. Create a *uid* or *gid* for HAB user. Habitat automatically sets a uid and gid for the HAB user. You can override it if required or you can leave the field *habitat_uid_gid=""* blank. _optional_
+
+1. Type the command, `./scripts/credentials set postgresql -auto` and press **Enter**. This command rotates the credentials for Postgresql.
+
+1. Type the command, `./scripts/credentials set elasticsearch -auto` and press **Enter**. This command rotates the credentials for ElasticSearch.
+
+1. Type the command, `chef-automate test -full` and press **Enter**. This command runs smoke tests on the setup.
+
+<!-- The default location for the secrets key and secret storage is set in the config file. The default location for the key is /etc/chef-automate/secrets.key and the secret store file is in /hab/a2_deploy_workspace/secrets.json -->
+
+## Clear the Bare Metal Infrastructure
```

Comment:
Same as Aws
----------------------------
# HOW TO DESTROY INFRA

# SCENARIO 1

If you are running chef-automate provision-infra and in between anything occure and provision get stopped then run below command to destroy that half part to start provision again.

`for i in 1;do i=$PWD;cd /hab/a2_deploy_workspace/terraform/;terraform destroy -state=/hab/a2_deploy_workspace/terraform/destroy/aws/terraform.tfstate;cd $i;done`

# SCENARIO 2

If you have successfully done provision-infra and after that if you want to destroy infra then run below command in below order.

`for i in 1;do i=$PWD;cd /hab/a2_deploy_workspace/terraform/destroy/aws/;terraform init;cd $i;done`

`for i in 1;do i=$PWD;cd /hab/a2_deploy_workspace/terraform/destroy/aws/;terraform destroy;cd $i;done`

# SCENARIO 3

If you've completed `chef-automate deploy` and want to destroy deploy part then run below command. Remember destroy of deployment will not remove any kind of configuration from remote server but it will do terraform taint so that next time all the configuration things will be started again.

`for i in 1;do i=$PWD;cd /hab/a2_deploy_workspace/terraform/;terraform destroy;cd $i;done`

# SCENARIO 4

If you have finished deployment and want destroy all the instances of whole infra then run below commnd.

`for i in 1;do i=$PWD;cd /hab/a2_deploy_workspace/terraform/destroy/aws/;terraform init;cd $i;done`
 
`for i in 1;do i=$PWD;cd /hab/a2_deploy_workspace/terraform/destroy/aws/;terraform destroy;cd $i;done